### PR TITLE
docs: add instruction of using gitpod

### PR DESCRIPTION
**Pull request by shootermv**
# Description
Instruction of running a project at gitpod.io remote editor

## screenshots
Not relevant

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -26,27 +29,26 @@ Follow these steps to run the project on your local machine:
...
-   After all dependencies are installed, you can start the project by running:
-   ```bash
-   npm start
-   ```
-   The project should now be running on your local machine. Open your web browser and navigate to http://localhost:3000 to view the project.
+    After all dependencies are installed, you can start the project by running:
+    ```bash
+    yarn start
+    ```
+    The project should now be running on your local machine. Open your web browser and navigate to http://localhost:3000 to view the project.
```
Comment (NoamGaash) [inline CONTRIBUTING.md:50]: seems like an accident

Comment (shootermv) [reply CONTRIBUTING.md:50]: Yes,Lot of things changed suddenly...
I will close this pr and open new one

Comment (NoamGaash) [reply CONTRIBUTING.md]: no need - you can revert the last commit and merge from main

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -13,7 +13,15 @@ Some of them marked with `help wanted` or `good first issue` - can be good for a
 
 Also! you can look at the project and if you find some point that can be improved (it can be unclear documentation as well as UI or backend problem) - you wellcome to add issue to the project issues (which considers a contribution too)
 
-# Running the Project Locally
+# Running the Project
```
Comment (NoamGaash) [inline CONTRIBUTING.md:16]:
```suggestion
# Remotely running the project
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -13,7 +13,15 @@ Some of them marked with `help wanted` or `good first issue` - can be good for a
...
+
+For start working on the project you can use any of options from the below:
+
+- You can set up project locally. This is **highly recommended** for regular/repeat contributions. This setup allows you to work and test your changes.
+- Use Gitpod, a free online dev environment. Clicking the link below will start a ready-to-code dev environment in your browser. It only takes a few minutes.
+
+Click [here](https://gitpod.io/#https://github.com/hasadna/open-bus-map-search) to
+open project in Gitpod
```
Comment (NoamGaash) [inline CONTRIBUTING.md:25]:
```suggestion

# Running the project on a local environment
```

**Review by NoamGaash — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (NoamGaash) [reply CONTRIBUTING.md]: I see you made it work in the same PR. well done :muscle:

**Pull request merged**